src/lib/expansion-panel/expansion-panel.ts

```diff
@@ -11,6 +11,7 @@ export interface IExpansionPanelComponent extends IBaseComponent {
   open: boolean;
   orientation: ExpansionPanelOrientation;
   animationType: ExpansionPanelAnimationType;
+  targetButton: string;
```

Comment:
Let's rename the `targetButton` property and `target-button` attribute to something like `trigger`? Open to other suggestions.